### ISSUE TITLE
 CP Selection is not working when drawing multiple NURBS

### ISSUE THREAD

**Issue by kdaic** [mi-lib/roki-gl#14]
I tried to implement a sample multi_nurbs_2d_test.c in branch [fix/cp_nurbs3d_selection](https://github.com/mi-lib/roki-gl/blob/fix/cp_nurbs3d_selection/example/shape/multi_nurbs_2d_test.c) .
In this sample, CP Selection does not seem to be working.  

What could be the cause of this?
Let's investigate using this branch and add what we can infer,

**Comment by zhidao** [comment]
See:
https://github.com/mi-lib/roki-gl/blob/main/example/shape/nurbs_2d_test.c

**Comment by kdaic** [comment]
I merged the main branch and checked the sample nurbs_2d_test.c.

Hmm... I see, it was my mistake.

In my sample multi_nurbs_test.c,
After checking the CP name and the NURBS curve name from the SelectionBuffer, only the CP name was being obtained. The name of the NURBS curve could not be obtained.

I also got the names of the curves, like in the updated nurbs_2d_test.c and it worked as expected.

Thank you for showing me how to use it.
